app/src/main/java/br/com/nespressohelper/daitan/nespressohelper/dbCreate.java

```diff
@@ -0,0 +1,17 @@
+package br.com.nespressohelper.daitan.nespressohelper;
+
+import android.database.sqlite.SQLiteDatabase;
+
+/**
+ * Created by estrela on 12/21/17.
+ */
+
+public class dbCreate {
+
+    public void createTable() {
+        SQLiteDatabase database = db.getInstance().getWritableDatabase();
+        String cols = "(NAME TEXT, CAPSULES INTEGER, BARS1 INTEGER, BARS2 INTEGER, DESC TEXT, KEY_IMAGE BLOB, unique(NAME, CAPSULES, BARS1, BARS2, DESC, KEY_IMAGE) )";
```

Comment:
Why use a unique constraint on all of the fields instead of a primary key?
If you're going to query this DB a properly identified primary key field could speed up the search